packages/mqttsuite/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/SNodeC/mqttsuite
+TERMUX_PKG_DESCRIPTION="MQTTSuite: A lightweight MQTT Integration System"
+TERMUX_PKG_LICENSE="MIT OR LGPL-3.0-or-later"
+TERMUX_PKG_LICENSE_FILE="LICENSE"
+TERMUX_PKG_MAINTAINER="VolkerChristian"
+TERMUX_PKG_VERSION=1.0
+TERMUX_PKG_SRCURL="git+https://github.com/SNodeC/mqttsuite.git"
+TERMUX_PKG_DEPENDS="fmt,snode.c"
```

Comment:
Dependency lists usually include a space between entries for readability,
though this isn't strictly required.
```suggestion
TERMUX_PKG_DEPENDS="fmt, snode.c"
```